extension/Cargo.toml

```diff
@@ -8,18 +8,18 @@ crate-type = ["cdylib"]
 
 [features]
 default = ["pg14"]
-pg12 = ["pgx/pg12", "pgx-tests/pg12"]
-pg13 = ["pgx/pg13", "pgx-tests/pg13"]
-pg14 = ["pgx/pg14", "pgx-tests/pg14"]
-pg15 = ["pgx/pg15", "pgx-tests/pg15"]
+pg12 = ["pgrx/pg12", "pgrx-tests/pg12"]
+pg13 = ["pgrx/pg13", "pgrx-tests/pg13"]
+pg14 = ["pgrx/pg14", "pgrx-tests/pg14"]
+pg15 = ["pgrx/pg15", "pgrx-tests/pg15"]
 pg_test = ["approx"]
 
 [dependencies]
-# Keep synchronized with `cargo install --version N.N.N cargo-pgx` in Readme.md and docker/ci/Dockerfile
-# Also `pgx-tests` down below in `dev-dependencies`.
-pgx = "=0.7.1"
-pgx-macros = "=0.7.1"
-pgx-sql-entity-graph = "=0.7.1"
+# Keep synchronized with `cargo install --version N.N.N cargo-pgrx` in Readme.md and docker/ci/Dockerfile
+# Also `pgrx-tests` down below in `dev-dependencies`.
+pgrx = "=0.9.1"
+pgrx-macros = "=0.9.1"
+pgrx-sql-entity-graph = "=0.9.1"
```

Comment:
```suggestion
pgrx = "=0.9.7"
pgrx-macros = "=0.9.7"
pgrx-sql-entity-graph = "=0.9.7"
```

Reply:
Thanks! I updated the pgrx version to 0.9.7 (unfortunately I couldn't do it by applying your suggestions directly on GitHub since the lockfile also needed to be updated).